packages/accessibility/src/AccessibilityManager.ts

```diff
@@ -32,16 +34,30 @@ const DIV_HOOK_ZINDEX = 2;
  */
 export class AccessibilityManager
 {
+    public readonly isActive: boolean;
```

Comment:
@GoodBoyDigital @Zyie lets make those two readonly prop + private field.

Reply:
Private + getter is more usable in this case becouse ot reassing a few times.

Reply:
Ok I've changed both of these to be private and have a getter.